Add note about coupled-motor usage in differentials (#3532)
cyberbotics/webots: docs/reference/motor.md

> **Note**: The motors are *logically* coupled together, not *mechanically*.
If one of the motors is physically blocked, the others are in no way affected by it.

> This provides a useful side-effect: when used in force-control mode, coupled-motor allows to easily simulate a mechanical differential.

Reviewer: ```suggestion
This provides a useful side-effect: when used in force-control mode, coupled-motors allow to easily simulate a mechanical differential.
```

The role of a differential is to change the speed of the wheel relatively to each other.
But also, it splits *equally* the motor torque to each wheel.

> This is why it's only needed to apply the same torque on multiple (coupled) motors, and the physic engine will adapt the speeds accordingly.

Reviewer: ```suggestion
Therefore it suffices to apply the same torque on multiple (coupled) motors, and the physics engine will adapt the speeds accordingly.
```

It works for a regular car as well as for a 4x4 vehicle, as long as they have 3 differentials (front, rear, and central).

> Using `multiplier` here also makes sense, because some differentials don't split in half: sometimes a central differential splits 40%-60% to get more torque to the rear wheels.

Reviewer: ```suggestion
Using `multiplier` here also makes sense, because some differentials do not split in half: sometimes a central differential splits 40%-60% to get more torque to the rear wheels.
```

Reviewer: No contractions, ideally.

This parameter doesn't depend on the actual speed and characteristics of the wheels, it's only a mechanical setting.

> **Note**: Although any among the coupled motors can be controlled, commands should be given to just one among them at any given time in order to avoid confusion or conflicts.

> For instance, it isn't possible to do Position Control for one motor and Velocity Control another at the same time.

Reviewer: ```suggestion
For instance, it is not possible to do Position Control for one motor and Velocity Control another at the same time.
```

Reviewer: Since we're removing contractions, might as well remove this one.
